fvm/fvm_test.go

```diff
@@ -3013,4 +3013,65 @@ func TestEVM(t *testing.T) {
 			))
 		}),
 	)
+
+	t.Run("deploy contract code", newVMTest().
+		withBootstrapProcedureOptions(fvm.WithSetupEVMEnabled(true)).
+		// we keep this dissabled during bootstrap and later overwrite in the test for test transaction
```

Comment:
you don't need to do this anymore right.

Reply:
Right, thanks.